### DISCUSSION TITLE
Jenkins pipeline failing with error while practicing terraform locals.

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [learndevopsonline/learndevopsonline#1611] [Q&A] [answered]
Hi Team,

Im getting below error while practicing terraform locals in Roboshop-terraform project.
My roboshop terraform pipeline is failing with below error.

<img width="682" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/113025693/e29bd399-d49d-40b0-9938-f87d1ff79d0e">
<img width="619" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/113025693/209048ab-c7bb-4e07-b012-a816a20d5360">

i think the error could be on formatting of the below line in locals.tf of terraform project.

<img width="769" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/113025693/25518812-48f7-4ac9-acaf-9b82f0b1b906">

i have enabled wrap statement in IntelliJ, but my pipeline still failing.

Is there any way to use the lengthy lines into multiple lines without breaking the syntax.

i look forward for your response.

Thank you

**Comment by sreedharm07** [comment]
Hi @LokeshNagireddy 
the command should be in single line but in your code comand in two lines thats why error is happening 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/3628bbbf-8de8-4f9e-aa4a-2bf30f63b2aa)


right click on numbers column in intilij you can see soft wrap click on it and after soft wrap make code in single line.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/19c85d8c-7990-433c-9f72-73612c7da02c)

**Reply by LokeshNagireddy** [reply]
Hi,
Thanks for your response.
when the error First occurred, its in a single line, then i tried to solve the issue by trying different approaches, one of that is Hard wrap.
As you suggested i tried soft wrap, but the issue was not solved.
I have increased the Hard wrap at value to 160, decreased the font size etc and tried. but got the error asusual.

<img width="658" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/113025693/0a424cff-a430-4c1d-8c2d-1bb4069679ed">

This error is happening for apps only (like frontend, payment, cart etc) due to the ansible command is written only for apps.

@r-devops , may you please help me in this issue. Soft wrap also not working.

i look forward for your response.

**Reply by sreedharm07** [reply]
can you please turn on your workstation and ping ip here

**Reply by LokeshNagireddy** [reply]
Hi Sreedharm,

I have started my workstation. please find the details below.

<img width="742" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/113025693/3c5e5fee-ae77-4e35-a068-7e457e632abc">

**Reply by sreedharm07** [reply, marked as answer]
hi @LokeshNagireddy 
in your locals.tf ansible command was wrong its supposed to be **ansible-pull**, but in you given **ansible pull** please change it and re-run it will works
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/6c5ebbfa-cc96-4f1e-9c9a-25a48bb83930)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/9eef8c7c-0e23-4981-942c-727d3f183a65)

**Reply by LokeshNagireddy** [reply]
Thanks alot, it worked...

**Comment by LokeshNagireddy** [comment]
Issue solved.. missing hyphen (-) caused the issue.
